packages/react-core/src/demos/PrimaryDetail.md

```diff
@@ -1452,6 +1446,38 @@ class PrimaryDetailCardView extends React.Component {
       swaggerIcon
     };
 
+    const cardHeaderActions = (
```

Comment:
Since the dropdown for these actions needs the `key` from the `filtered.map` on line 1457, could we just inline this oni line 1495 or make this a function that returns the actions? The demo in the surge currently throws a reference error, "key is not defined"